themes/default/content/registry/packages/docker/_index.md

```diff
@@ -4,7 +4,7 @@ meta_desc: Provides an overview of the Docker Provider for Pulumi.
 layout: overview
 ---
 
-The Docker provider for Pulumi can be used to provision any of the resources available in [Docker](https://www.docker.com/).
+The Pulumi Docker Provider can be used to provision and interact with any of the resources available in [Docker](https://www.docker.com/) including containers, images, networks, volumes and more.
```

Comment:
@mnlumi are we capitalizing provider now? i think there may be some standardization to do across things if so

Reply:
Yes, I had some discussions around capitalization with @meagancojocar. When we are referencing a official Pulumi provider, provider should be capitalized as that is the proper noun/name of it. Standardization will continue to be a WIP.